lib/inspec/reporters/automate.rb

```diff
@@ -0,0 +1,76 @@
+# encoding: utf-8
+
+require 'json'
+require 'net/http'
+
+module Inspec::Reporters
+  class Automate < Json
+    def initialize(config)
+      super(config)
+
+      # default to using no ssl for sending reports
+      @config['use_ssl'] = @config['use_ssl'] || false
```

Comment:
Would `@config['use_ssl'] ||= false` work here?

Reply:
I think `use_ssl` is not precise, because we use ssl in any case. The only difference is that in one case we validate the certificate and in the other we don't. Maybe `@config['verify_ssl'] = @config['verify_ssl'] || false`